src/Upgrade/Application/Strategy/Common/Step/OrmModelsRegenerateStep.php

```diff
@@ -112,26 +107,17 @@ public function handle(StepsResponseDto $stepsExecutionDto): StepsResponseDto
         foreach ($commandList as $command) {
             $response = $this->processRunner->run(explode(' ', $command), ['APPLICATION_ENV' => 'development']);
             if ($response->getExitCode()) {
-                $stepsExecutionDto->addOutputMessage($response->getErrorOutput());
+                $message = $response->getErrorOutput() ?: $response->getOutput();
```

Comment:
needs to include transfer generate console command before and after propel commands.

Reply:
The transfer generate command was added